CATALOG.md

```diff
@@ -65,7 +65,7 @@ Unique ID|http://test-network-function.com/testcases/access-control/pod-automoun
 Version|v1.0.0
 Description|http://test-network-function.com/testcases/access-control/pod-automount-service-account-token check that all pods under test have automountServiceAccountToken set to false
 Result Type|normative
-Suggested Remediation|check that pod has automountServiceAccountToken set to false or pod is attached to service account which has automountServiceAccountToken set to false
+Suggested Remediation|Check that pod has automountServiceAccountToken set to false or pod is attached to service account which has automountServiceAccountToken set to false
```

Comment:
Shouldn't the exception info also appear in the catalog ?

Reply:
I'll check on that!

Reply:
Fixed.